### ISSUE TITLE
TrueTransferOperator height and width

### ISSUE THREAD

**Issue by aschaf** [mfem/mfem#1435]
Hi,

is it by design that `TrueTransferOperator` does not have a height and a width? 

Thanks,
Andreas

**Comment by v-dobrev** [comment]
Sounds more like a bug to me. All `Operator`s should have these defined.

Just to narrow down where the issue is: what type of `TrueTransferOperator` do you have? The possible types are described here:
https://github.com/mfem/mfem/blob/99cf263ee490cd0431d7b2b798e3070c65a44d81/fem/fespace.hpp#L617-L632

**Comment by aschaf** [comment]
It is Operator::Hypre_ParCSR. 

I fixed it locally by changing
https://github.com/mfem/mfem/blob/7690eca85041e0b69afe81a063c71015184e82a3/fem/transfer.cpp#L520-L523
to
```
TrueTransferOperator::TrueTransferOperator(const
                                           ParFiniteElementSpace& lFESpace_,
                                           const ParFiniteElementSpace& hFESpace_)
   : Operator(hFESpace_.GlobalVSize(), lFESpace_.GlobalVSize()), lFESpace(lFESpace_), hFESpace(hFESpace_)
```

**Comment by aschaf** [comment]
I found the same issue also with using `L2ProjectionGridTransfer` in parallel,
https://github.com/mfem/mfem/blob/7690eca85041e0b69afe81a063c71015184e82a3/fem/fespace.hpp#L909
and
https://github.com/mfem/mfem/blob/7690eca85041e0b69afe81a063c71015184e82a3/fem/fespace.cpp#L2616-L2618

When applying the same fix as in my previous comment, it again seems to work.

**Comment by pazner** [comment]
I can make a small PR to fix these. I can either combine it with another minor bugfix PR (#1447) or make a new one. Thoughts?

**Comment by v-dobrev** [comment]
Note: the height and width of an `Operator` are the mpi-local sizes, so you want to use `fes.GetTrueVSize()`.

@pazner, if you want to add it to #1447, I'm fine with that. I'm also fine with a separate PR -- whatever you prefer.

**Comment by tzanio** [comment]
Thanks for reporting this @aschaf. Discussion continued in #1447.
